versions/3.1.0.md

```diff
@@ -197,6 +197,8 @@ Field Name | Type | Description
 <a name="oasSecurity"></a>security | [[Security Requirement Object](#securityRequirementObject)] | A declaration of which security mechanisms can be used across the API. The list of values includes alternative security requirement objects that can be used. Only one of the security requirement objects need to be satisfied to authorize a request. Individual operations can override this definition.
 <a name="oasTags"></a>tags | [[Tag Object](#tagObject)] | A list of tags used by the specification with additional metadata. The order of the tags can be used to reflect on their order by the parsing tools. Not all tags that are used by the [Operation Object](#operationObject) must be declared. The tags that are not declared MAY be organized randomly or based on the tools' logic. Each tag name in the list MUST be unique.
 <a name="oasExternalDocs"></a>externalDocs | [External Documentation Object](#externalDocumentationObject) | Additional external documentation.
+<a name="oasWebhooks"></a>webhooks | Map[`string`, [Path Item Object](#pathItemObject)] | The incoming webhooks that may be received as part of this API. The key name is a unique string to refer to each webhook, while the Path Item Object describes a request that may be initiated by the API provider and the expected responses. An [example](../examples/v3.1/webhook-example.yaml) is available.
```

Comment:
- I'd like to see this entry right after the `paths`. 
- `MAY`
- Not a huge fan of the link to the external example. I realize we also have it in the callbacks, not sure we should have it there either.
- It might also prove useful to further explain that this can be used to describe out of bound webhook registration, the relationship with callbacks and that these are meant to be implemented by the API consumer. It might be clear to us as we've all discussed it in the past, not sure a first time reader will get it as well.

Reply:
I've made the changes you suggest and expanded more on the wording, would you take another look? I have left the external example in since I think it is useful to have one somewhere but I'm open to suggestions about where!